students/17th/team1/jiyoonlee/user/views.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from django.http import JsonResponse
+from django.views import View
+
+from .models import User
+
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            mobile_number = data['mobile_number']
+            email         = data['email']
+            full_name     = data['full_name']
+            username      = data['username']
+            password      = data['password']
+
+            if mobile_number is None and email is None:
+                return JsonResponse(
+                    {'message': '휴대폰 번호나 이메일을 입력하세요.'}, 
+                    status=400
+                )
+
+            if email is not None:
+                if '@' not in email or '.' not in email:
+                    return JsonResponse(
+                        {'message': '이메일 형식을 확인하세요.'}, 
+                        status=400
+                    )
+            
+            if len(password) < 8:
```

Comment:
```suggestion
            if len(password) < MINIMUN_PASSWORD_LENGTH:
```
8과 같은 비교값은 하드코딩하지 말고, 상수화 시켜서 사용하는게 좋습니다.
상수선언은 코드상단 임포트 아래 혹은 함수의 바깥에 선언해주세요.

Reply:
수정했습니다